docs/v2/openapi.documented.yml

```
@@ -2298,6 +2074,83 @@ paths:
+                #/components/schemas/Body_create_messages_with_file_v2_workspaces__workspace_id__sessions__session_id__messages_upload_post
+      responses:
+        '200':
+          description: Successful Response
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Message'
+                title: >-
+                  Response Create Messages With File V2 Workspaces  Workspace Id  Sessions  Session Id 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)